src/swap/letsexchange.js

```diff
@@ -0,0 +1,262 @@
+    ): Promise<EdgeSwapQuote> {
+      checkInvalidCodes(INVALID_CURRENCY_CODES, request, swapInfo)
+
+      // Grab addresses:
+      const [fromAddress, toAddress] = await Promise.all([
+        getAddress(request.fromWallet, request.fromCurrencyCode),
+        getAddress(request.toWallet, request.toCurrencyCode)
+      ])
+
+      // Convert the native amount to a denomination:
+      const quoteAmount =
+        request.quoteFor === 'from'
+          ? await request.fromWallet.nativeToDenomination(
+              request.nativeAmount,
+              request.fromCurrencyCode
+            )
+          : await request.toWallet.nativeToDenomination(
+              request.nativeAmount,
+              request.toCurrencyCode
+            )
+
+      const {
+        fromMainnetCode,
+        toMainnetCode
+      } = getCodesWithMainnetTranscription(request, MAINNET_CODE_TRANSCRIPTION)
+
+      // Swap the currencies if we need a reverse quote:
+      const quoteParams = {
+        from: request.fromCurrencyCode,
+        to: request.toCurrencyCode,
+        network_from: fromMainnetCode,
+        network_to: toMainnetCode,
+        amount: quoteAmount
+      }
+
+      log('quoteParams:', quoteParams)
+
+      // Calculate the amounts:
+      let fromAmount, fromNativeAmount, toNativeAmount, reply
+      if (request.quoteFor === 'from') {
+        reply = await call(uri + 'info', request, {
+          params: quoteParams
+        })
+        fromAmount = quoteAmount
+        fromNativeAmount = request.nativeAmount
+        toNativeAmount = await request.toWallet.denominationToNative(
+          reply.amount.toString(),
+          request.toCurrencyCode
+        )
+      } else {
+        reply = await call(uri + 'info-revert', request, {
+          params: quoteParams
+        })
+        fromAmount = reply.amount
+        fromNativeAmount = await request.fromWallet.denominationToNative(
+          fromAmount.toString(),
+          request.fromCurrencyCode
+        )
+        toNativeAmount = request.nativeAmount
+      }
+
+      // Check the minimum:
+      const nativeMin = await request.fromWallet.denominationToNative(
+        reply.min_amount,
+        request.fromCurrencyCode
+      )
+
+      if (lt(fromNativeAmount, nativeMin)) {
+        throw new SwapBelowLimitError(swapInfo, nativeMin)
+      }
+
+      const { promoCode } = opts
+      const sendReply = await call(uri + 'transaction', request, {
+        params: {
+          deposit_amount: fromAmount,
+          coin_from: request.fromCurrencyCode,
+          coin_to: request.toCurrencyCode,
+          network_from: fromMainnetCode,
+          network_to: toMainnetCode,
+          withdrawal: toAddress,
+          return: fromAddress,
+          return_extra_id: null,
+          withdrawal_extra_id: null,
+          affiliate_id: initOptions.apiKey,
+          promocode: promoCode != null ? promoCode : '',
+          type: 'edge',
+          float: true,
```

Comment:
I previously requested that the plugin attempt both types of trade, fixed and floating, as this will maximize the opportunities for LetsExchange to be presented to a user. If you're only going to go with a single type I highly recommend doing a fixed rate since the app will always prioritize fixed rate trades over a floating rate.